integrations/extensions/docs/elasticsearch-install-and-setup/how_to_use_web_crawler_in_elasticsearch.md

```diff
@@ -0,0 +1,315 @@
+    -v "${ES_CACERT_FOLDER}:/usr/share/enterprise-search/es-config:ro" \
+    --interactive \
+    --tty \
+    --rm \
+    --env "secret_management.encryption_keys=[${ENCRYPT_KEY}]" \
+    --env "allow_es_settings_modification=true" \
+    --env "elasticsearch.host=${ES_URL}" \
+    --env "elasticsearch.username=${ES_USER}" \
+    --env "elasticsearch.password=${ES_PASSWORD}" \
+    --env "elasticsearch.ssl.enabled=true" \
+    --env "elasticsearch.ssl.certificate_authority=/usr/share/enterprise-search/es-config/${ES_CACERT_NAME}" \
+    --env "kibana.external_url=http://kibana:5601" \
+    "docker.elastic.co/enterprise-search/enterprise-search:${ES_VERSION}"
+    ```
+* Verify it 
+  * Open http://localhost:5601 in your browser and log into Kibana using your Elasticsearch username and password.
+  * Navigate to the Search Overview page http://localhost:5601/app/enterprise_search/overview.
+  * If you can see `Web Crawler` available as an option to ingest content, your Enterprise Search has been set up successfully.
+
+### Set up Enterprise Search for watsonx Discovery on CloudPak
+Coming soon...
+
+## Step 2: Create and configure a web crawler in Elasticsearch 
+* In Kibana, you can find the Web Crawler option on Search Overview http://localhost:5601/app/enterprise_search/overview. 
+Click on `Start`, and follow the steps to create a Web Crawler index. 
+
+
+* On the `Manage Domains` tab, add a domain, for example, `https://www.nationalparks.org`.
+
+
+* Add an entry point under the domain `Entry points` tab, for example, `https://www.nationalparks.org/explore/parks`
+
+
+* From the domain `Crawl Rules` tab, add two additional rules like below:  
+  <img src="assets/add_crawl_rules_for_web_crawler.png" width="802" height="309" />  
+  NOTE: The two rules tell the web crawler to only crawl URLs that begin with https://www.nationalparks.org/explore/parks. 
+  Learn more about crawl rules from [here](https://www.elastic.co/guide/en/app-search/8.10/web-crawler-reference.html#web-crawler-reference-crawl-rule)
+
+
+* You can now try the web crawler by clicking on `Crawl` in the upper right corner.  
+  NOTE: The web crawler uses a default ingest pipeline to extract web content and store them as documents in the index. 
+  The default ingest pipeline doesn't use chunking or semantic search such as ELSER to process the documents. To enable chunking 
+  and ELSER for your web crawler, you will need to finish [Step 3](#step-3-create-an-elser-ingest-pipeline-with-a-chunking-processor) 
+  first to build a custom ingest pipeline with a chunking processor, and then start the web crawler. 
+
+## Step 3: Build an ELSER ingest pipeline with a chunking processor
+To use ELSER for text expansion queries on chunked texts, you need to build an ingest pipeline with a chunking processor that uses the ELSER model.
+
+
+
+### Update the index mappings of your web crawler
+```shell
+curl -X PUT "${ES_URL}/<your-web-crawler-index-name>/_mapping?pretty" -k \
```

Comment:
Can we put `<your-web-crawler-index-name>` into an environment variable so it is possible to just copy/paste this curl command into a shell?